Multisig/UI/App/AppViewModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AppViewModel.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 02.10.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class AppViewModel {
+    static var shared = AppViewModel()
+
+    var coins = CoinBalancesModel()
+    var collectibles = CollectibleBalancesModel()
+    var transactions = LoadingTransactionListViewModel()
+    var safeSettings = LoadingSafeSettingsViewModel()
+    private var txDetailsPool = [String: LoadingTransactionDetailsViewModel]()
```

Comment:
will we have the same in future for collectables, balances ?

Reply:
might be.. I don't like it because that's basically eternal retain, but I couldn't come up with another choice for now